application/src/main/java/com/softserve/lv460/application/controller/LocationController.java

```diff
@@ -0,0 +1,43 @@
+package com.softserve.lv460.application.controller;
+
+import com.softserve.lv460.application.dto.location.LocationRequest;
+import com.softserve.lv460.application.dto.location.LocationResponse;
+import com.softserve.lv460.application.service.LocationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/location")
+@CrossOrigin
+public class LocationController {
+  @Autowired
+  private LocationService locationService;
+
+  @PostMapping
+  public void create(@RequestBody LocationRequest request) {
+    locationService.create(request);
+  }
+
+  @GetMapping
+  public List<LocationResponse> findAll() {
+    return locationService.findAll();
+  }
+
+  @PutMapping
+  public void update(@RequestParam Long id, @RequestBody LocationRequest request) {
```

Comment:
it's better to have path variable instead of requestParam, so you'll have /locations/{location_id}